### ISSUE TITLE
An exception occured: Property "claimValue" cannot be found.

### ISSUE THREAD

**Issue by davidjhole** [kayasax/EasyPIM#54]
Hello, trying out your script for the fist time, running the lates 1.6.3 version and get the error below, if I go back to 1.6.2 it works fine.

`Get-PIMAzureResourcePolicy -TenantID $tenantID -SubscriptionId $subscriptionID -rolename "reader"`

Verbose output

```
PS C:\_repo\NotTracked\Test-PIM> Get-PIMAzureResourcePolicy -TenantID $tenantID -SubscriptionId $subscriptionID -rolename "reader" -Verbose
VERBOSE: Get-PIMAzureResourcePolicy start with parameters: subscription => ********-****-****-****-************, rolename=> reader
VERBOSE:  #1 Get role definition for the role reader assignable at the scope subscriptions/********-****-****-****-************ at https://management.azure.com/subscriptions/********-****-****-****-************/providers/Microsoft.Authorization/roleDefinitions?api-version=2022-04-01&$filter=roleName eq 'reader'
VERBOSE: 
>> request body: 
VERBOSE: request URI : https://management.azure.com/subscriptions/********-****-****-****-************/providers/Microsoft.Authorization/roleDefinitions?api-version=2022-04-01&$filter=roleName eq 'reader'
VERBOSE: >> Getting access token
VERBOSE: >> RodeId = /subscriptions/********-****-****-****-************/providers/Microsoft.Authorization/roleDefinitions/acdd72a7-3385-48ef-bd42-f606fba81ae7
VERBOSE:  #2 Get the Assignment for reader at https://management.azure.com/subscriptions/********-****-****-****-************/providers/Microsoft.Authorization/roleManagementPolicyAssignments?api-version=2020-10-01&$filter=roleDefinitionId eq '/subscriptions/********-****-****-****-************/providers/Microsoft.Authorization/roleDefinitions/acdd72a7-3385-48ef-bd42-f606fba81ae7'
VERBOSE: 
>> request body: 
VERBOSE: request URI : https://management.azure.com/subscriptions/********-****-****-****-************/providers/Microsoft.Authorization/roleManagementPolicyAssignments?api-version=2020-10-01&$filter=roleDefinitionId eq '/subscriptions/********-****-****-****-************/providers/Microsoft.Authorization/roleDefinitions/acdd72a7-3385-48ef-bd42-f606fba81ae7'
VERBOSE: >> Getting access token
VERBOSE: >> policy ID = /subscriptions/********-****-****-****-************/providers/Microsoft.Authorization/roleManagementPolicies/bd88118e-e6ac-4e64-bf8c-32004f8d0fe1
VERBOSE:  #3 get role policy at https://management.azure.com//subscriptions/********-****-****-****-************/providers/Microsoft.Authorization/roleManagementPolicies/bd88118e-e6ac-4e64-bf8c-32004f8d0fe1/?api-version=2020-10-01
VERBOSE: 
>> request body: 
VERBOSE: request URI : https://management.azure.com//subscriptions/********-****-****-****-************/providers/Microsoft.Authorization/roleManagementPolicies/bd88118e-e6ac-4e64-bf8c-32004f8d0fe1/?api-version=2020-10-01
VERBOSE: >> Getting access token
VERBOSE: MyCatch function called
An exception occured: Property "claimValue" cannot be found.

Details:
Position: At C:\Users\uktpdhole\OneDrive\Documents\PowerShell\Modules\EasyPIM\1.6.3\EasyPIM.psm1:128 char:141
+ … cationContext_EndUser_Assignment" } |Select-Object -expand claimValue
+                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VERBOSE: MyCatch function called
An exception occured: Error, script did not terminate gracefuly

Details:
Position: At C:\Users\uktpdhole\OneDrive\Documents\PowerShell\Modules\EasyPIM\1.6.3\EasyPIM.psm1:1120 char:5
+     throw "Error, script did not terminate gracefuly" #fix issue #40
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Exception: Error, script did not terminate gracefuly
```

**Comment by kayasax** [comment]
Thank you @davidjhole for reporting this. It seems the issue is within PowerShell 5.x, I will look into this, you can use posh.exe in the meantime.

**Comment by davidjhole** [comment]
I am using posh, so could be some version miss match, just let me know if you need any more info

Name                           Value
----                           -----
PSVersion                      7.3.12
PSEdition                      Core
GitCommitId                    7.3.12
OS                             Microsoft Windows 10.0.22631
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

**Comment by kayasax** [comment]
Indeed just tried with PS V5 and it's working on my side.
I'm not able to reproduce...
Do you have same issue with another role? 
With Get-PIMEntraRolePolicy -TenantID $tenantID  -rolename "testrole" ?
and get-pimgrouppolicy -tenantID $tenantID -groupID $gid -type owner ?

**Comment by kayasax** [comment]
Fixed in V1.6.4

**Comment by davidjhole** [comment]
Thanks it's working now 👍
